### PR TITLE
Issue #5572 - Bug fixes for Google ReCaptcha 2.0 update

### PR DESCRIPTION
**Pull request by emeraldarcher**
This contains the remaining bug fixes for the update to Google ReCaptcha in Orchard. The update was merged in issues 5331 and 5332.

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     var element = (ReCaptcha)Model.Element;
     var publicKey = (string)Model.PublicKey;
 }
-@if (!String.IsNullOrWhiteSpace(publicKey)) {
+@if (HasText(publicKey)) {
```
Comment (sfmskywalker) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml]: I wonder why you're using `HasText` on strings? Isn't that intended to be used on objects other than strings?

Comment (emeraldarcher) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml]: I was told to change from IsNullOrWhitespace() to HasText() by Jetski5822.
I can change it back though if you'd like.

Comment (sfmskywalker) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml]: I see. @Jetski5822 what's the thinking behind `HasText`, do you know?

Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml]: We talked about it during the triage, and I think I commented on the "request" from Nick. @sfmskywalker  is right, you can just use `String.IsNullOrEmptyWhitespace` here. But `HasText` begin technically correct we didn't bother accepting the PR.

Comment (Jetski5822) [inline src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml]: Huh?

**Comment by sebastienros** [comment]
Feel free to create another PR to remove the HasText

**Pull request merged**